api/net/Areas/Admin/Controllers/IngestController.cs

```diff
@@ -165,6 +165,29 @@ public async Task<IActionResult> UpdateIngestEnabledStatusAsync(int id, bool new
         return new JsonResult(new IngestModel(model, _serializerOptions));
     }
 
+    /// <summary>
+    /// Update ingest with the specified 'id'.
+    /// </summary>
+    /// <param name="id"></param>
+    /// <returns></returns>
+    [HttpPut("{id}/reset")]
```

Comment:
nit picking here, but could we change out the name `reset` here for `resetfailures`.  Contextually that's all we are resetting, not the whole Ingest.  This would apply to the method name and also to the `useApiAdminIngests`